content/60.methods.md

```diff
@@ -1,50 +1,101 @@
+However, these tools are not typically associated with biomedical research, yet synthesis and discussion of the emerging literature by biomedical scientists and clinicians would be expected to provide novel insights into how various areas of COVID-19 research intersect.
+
+Here, we describe efforts to develop Manubot to handle an extreme case study in open, collaborative scientific writing: developing reviews of eight topics pertaining to the COVID-19 pandemic.
+The goal of the COVID-19 Collaborative Review community was to consolidate information about the virus in the context of related viruses and to synthesize rapidly emerging literature centered on the diagnosis and treatment of COVID-19.
+Manubot provided the infrastructure to manage contributions from the community and create a living, scholarly document.
 We designed software to generate figures that automatically update using external data sources. <!-- TODO: replace figure reference with manuscript reference or reference figure conditionally based on template variable [, such as Figure @fig:csse-deaths,] -->
-Our primary goal is to sort and distill informative content out of the overwhelming flood of information [@doi:10.1038/s42254-020-0175-7] and help the broader scientific community become more conversant on this critical subject.
+Our primary goal was to provide the backend for scientists to sort and distill informative content out of the overwhelming flood of information [@doi:10.1038/s42254-020-0175-7] and help the broader scientific community become more conversant on this critical subject.
 Thus, our approach has been to develop a real-time, collaborative effort that welcomes submissions from scientists worldwide into this ongoing effort.
-This document represents the first snapshot, which aims to reflect the state of the field as of October, 2020.
-We plan to refine and expand this document until technologies to mitigate the pandemic are widely available.
+This case study demonstrates the value of open collaborative writing tools such as Manubot to emerging crises such as COVID-19.
+By allowing us to keep a record of the evolution of information over time and to create figures and in-line text that autoupdated alongside the state of the world, it revealed the particular value that Manubot holds for managing a rapid changes in scientific thought.
 
-In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becomes available, this project is an open, collaborative effort that invited contributions from the scientific community broadly, similar to previous efforts to develop collaborative reviews [@doi:10.1098/rsif.2017.0387; @url:https://greenelab.github.io/deep-review].
+### Methods
+
+#### Description of the Problem
+
+<!--To Do: Plot the number of pubs in CORD-19 over time?-->
```

Comment:
A plot like this - growth of preprints and/or published manuscripts - would be nice.  It could be the first part of Results instead of Methods.